config/database.yml

```diff
@@ -27,7 +27,7 @@ development:
   # To create additional roles in PostgreSQL see `$ createuser --help`.
   # When left blank, PostgreSQL will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: blog_app
+  #username: my_blog_app
 
   # The password associated with the PostgreSQL role (username).
```

Comment:
- [ ] Dear @ulises2607, Just as mentioned by the previouse reviewer some of the tests are still falling. I can see most of the errors are related to the route handler. as the error states 

```ruby
     ActionController::UrlGenerationError:
       No route matches {:action=>"index", :controller=>"posts", :user_id=>nil}, possible unmatched constraints: [:user_id]
       Did you mean?  user_posts_url
```
Please look closer at the error message I have attached and try to address it. 

### The issue 
<img width="1680" alt="Screenshot 2023-11-17 at 9 21 44 at night" src="https://github.com/ulises2607/Blog-App/assets/69077061/7c0ac563-7a5f-43c2-a1bd-57cecc1b5370">

-[Optional] Alongside, I would love to remind you to work on the optional suggestion that the previous reviewer leaves.

Reply:
Please I need one meeting to explain this because I have lost a lot of time. I fixed it

Reply:
call me on zoom @ulises2607.

Reply:
Now please!! @ulises2607

Reply:
Ulises Cappa is inviting you to a scheduled Zoom meeting.

Topic: Sala de reuniones personales de Ulises Cappa

Join Zoom Meeting
https://microverse.zoom.us/j/2347484106?pwd=c0srVVY0T0tkaFVmU2F6Z0FPZFBKdz09

Meeting ID: 234 748 4106
Passcode: 60FMQ4

---

One tap mobile
+16469313860,,2347484106# US
+16694449171,,2347484106# US

---

Dial by your location
• +1 646 931 3860 US
• +1 669 444 9171 US
• +1 669 900 6833 US (San Jose)
• +1 689 278 1000 US
• +1 719 359 4580 US
• +1 929 205 6099 US (New York)
• +1 253 205 0468 US
• +1 253 215 8782 US (Tacoma)
• +1 301 715 8592 US (Washington DC)
• +1 305 224 1968 US
• +1 309 205 3325 US
• +1 312 626 6799 US (Chicago)
• +1 346 248 7799 US (Houston)
• +1 360 209 5623 US
• +1 386 347 5053 US
• +1 507 473 4847 US
• +1 564 217 2000 US

Meeting ID: 234 748 4106

Find your local number: https://microverse.zoom.us/u/adnPAGlUgA

I don't know how can I do that!

Reply:
@TadesseDev